### ISSUE TITLE
Documents with namespaced ids cannot be edited via the desk tool's document lists

### ISSUE THREAD

**Issue by felthy** [sanity-io/sanity#1578]
**Describe the bug**

Since I upgraded from version `0.142.10` of the desk tool, documents with namespaced ids can no longer be edited by choosing them from a `documentTypeListItem`. They open a blank draft document instead.

The cause seems to be that the namespace is being stripped from the id - when following the steps to reproduce below, I can use the Network pane of Chrome's dev tools to see that the groq query being executed is `[*[_id in ["drafts.test-id","test-id"]][0...2]{_id,_rev,_type,_type}][0...1]` - the `namespace.` prefix is missing from the id parameters.

I can still edit the document if I open it directly via search - i.e. the route `http://localhost:3333/desk/edit/person/namespace.test-id` is still working.

**To Reproduce**

1. Run `sanity init` and choose the movie project template. Include a sampling of sci-fi movies.
2. Run `sanity documents create` and paste in the following document:
```json
{
  "_id": "namespace.test-id",
  "_type": "person",
  "name": "Test Person",
  "slug": {
    "_type": "slug",
    "current": "test-person"
  }
}
```
3. Run `sanity start` and visit http://localhost:3333
4. Click "Person" in the left hand column of the desk tool
5. "Test Person" should be the first person listed. Click that item.

**Expected behavior**

Instead of seeing the document's edit form, I see a blank draft document.

**Screenshots**

![image](https://user-images.githubusercontent.com/1940215/67836743-7ec81a80-fb41-11e9-8dd5-9f92faca1dd4.png)

![image](https://user-images.githubusercontent.com/1940215/67836773-9901f880-fb41-11e9-9058-6c06ab4f68f0.png)

**Which versions of Sanity are you using?**

```
@sanity/cli                0.144.0 (up to date)
@sanity/base               0.144.1 (up to date)
@sanity/components         0.144.2 (up to date)
@sanity/core               0.144.0 (up to date)
@sanity/default-layout     0.144.2 (up to date)
@sanity/default-login      0.144.0 (up to date)
@sanity/desk-tool          0.144.2 (up to date)
@sanity/google-maps-input  0.144.2 (up to date)
@sanity/vision             0.144.2 (up to date)
```

**What operating system are you using?**

macOS Catalina

**Which versions of Node.js / npm are you running?**

```
6.9.0
v10.16.3
```

**Comment by felthy** [comment]
The problem goes away if I make the following change to `package.json`:

```
    "@sanity/desk-tool": "0.142.10",
```

Then re-run `yarn` followed by `sanity start`.
Version `0.144.0` seems to have been the first version with this bug.

**Comment by felthy** [comment]
Appears to have been fixed in 0.144.3 by #1600.
